juju/juju
# juju offers command takes a model arg

## Description of change

The "juju offers" command is now a model command so that you can use -m to tell it to operate on a different model to the current one.

## QA steps

Run up a cmr scenario
switch to model hosting offers
juju offers -> see results
switch to another model
juju offers -m <model> -> results
juju offers -m <user>/<model> -> results
